addons/fastroping/CfgWeapons.hpp

```diff
@@ -0,0 +1,60 @@
+class CfgWeapons {
+    class InventoryItem_Base_F;
+	class ACE_ItemCore;
```

Comment:
Nope, there's definitely a tab here. Note​ how the indentation is weird in this file on Github.